File: KatlaSport.Services.Models/ProductManagement/Product.cs
namespace KatlaSport.Services.ProductManagement
{
    /// <summary>
    /// Represents a product.
    /// </summary>
    public class Product : ProductCategoryProductListItem
    {
        /// <summary>
        /// Gets or sets a parent product category identifier.
        /// </summary>
        public int CategoryId { get; set; }

        /// <summary>
        /// Gets or sets a product description.
        /// </summary>
        public string Description { get; set; }

        /// <summary>
        /// Gets or sets a product manufacturer code.
        /// </summary>
        public string ManufacturerCode { get; set; }

        /// <summary>
        /// Gets or sets a product price.
        /// </summary>
        public decimal Price { get; set; }
    }
}

File: KatlaSportNg/src/app/product-management/models/product.ts
export class Product {
    constructor(
        public id: number,
        public code: string,
        public name: string,
        public сategoryId: number,
        public description: string,
        public manufacturerCode: string,
        public price: number,
        public isDeleted: boolean,
        public lastUpdated: string
    ) { }
}
